bytecomp/bytelink.ml

```diff
@@ -324,7 +324,8 @@ let link_bytecode ?final_name tolink exec_name standalone =
        end;
        Bytesections.init_record outchan;
        (* The path to the bytecode interpreter (in use_runtime mode) *)
-       if String.length !Clflags.use_runtime > 0 then begin
+       if String.length !Clflags.use_runtime > 0 && not !Clflags.noruntime then
```

Comment:
This feels like the wrong place for this check - `!Clflags.use_runtime <> "" && !Clflags.noruntime` should be an error earlier.

Reply:
See my comment in the main PR thread.